src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -157,10 +159,10 @@ export default {
 		 */
 		closeActions(e) {
 			// Don't do anything if we leave towards a child element.
-			if (this.$refs.actions.$el.contains(e.relatedTarget)) {
+			if (this.$refs.actionsBreadcrumb.$el.contains(e.relatedTarget)) {
 				return
 			}
-			this.$refs.actions.opened = false
+			this.$refs.actionsBreadcrumb.$refs.actions.open = false
```

Comment:
Should be `this.$refs.actionsBreadcrumb.$refs.actions.opened = false`, right?
The other occurences as well.

Reply:
Fixed :)